src/mock/mockRoutes.ts

```diff
@@ -0,0 +1,20 @@
+import faker from "faker";
+
+//Custom Generator
+const routeGenerator = () => ({
+  name: faker.unique(faker.internet.url),
+  route: faker.unique(faker.system.filePath),
+  status: faker.datatype.boolean(),
+});
+
+//Generator Function
+const createRoutes = (numRoutes = 5) => {
+  return Array.from({ length: numRoutes }, routeGenerator);
+};
+
+// Mock data
```

Comment:
mock routes

Reply:
Are you talking about the comment here? Actually, I decided to divide mock data generation into three parts: Custom Generator, Generator Function, and Mock Data, so these comments actually tell the start of that part. As for the mock route, the variable name will suffice, I guess

Reply:
doesn't tell what the data/function is for